graphein/ml/datasets/torch_geometric_dataset.py

```diff
@@ -492,8 +513,10 @@ def processed_file_names(self) -> List[str]:
 
     @property
     def raw_dir(self) -> str:
-        if self.pdb_paths is not None:
-            return self.pdb_path  # replace raw dir with user local pdb_path
+        if self.pdb_paths:
```

Comment:
I actually think it would be useful to allow users to choose a path for `raw_dir` when initialising the Dataset objects.

Reply:
Yes, I agree. 

The question is if `os.listdir` in the func, and then the order of `self.structure` maybe hard to match the order of `graph_labels` and `node_labels`, since we match the `labels` by index of list, i guess. 
<img width="438" alt="image" src="https://user-images.githubusercontent.com/49584439/191144478-d8ebc806-1c63-46b0-95c9-bf95948eafad.png">

<img width="494" alt="image" src="https://user-images.githubusercontent.com/49584439/191144438-8d7f5649-174d-4a51-a7b4-0c940806fef9.png">

<img width="509" alt="image" src="https://user-images.githubusercontent.com/49584439/191144389-50435092-fc28-4d29-8aaf-d4f0ee6b5312.png">

I'm not sure about this, i prefer to dict, which key is the names like `{'10gs':0}` would be better than `{0:0}`.